taimen/config.json

```diff
@@ -89,21 +87,390 @@
       ],
       "product-bytecode": [],
       "forced-modules": [
+        "chre",
         "com.android.ims.rcsmanager",
-        "RcsService",
-        "netutils-wrapper-1.0"
+        "ese-ls-provision",
+        "ese-replay",
+        "libhidltransport.vendor",
+        "libhwbinder.vendor",
+        "libjson",
+        "libp61-jcop-kit.vendor",
+        "libsensorndkbridge",
+        "libtinyxml",
+        "netutils-wrapper-1.0",
+        "RcsService"
       ],
       "new-modules": [],
-      "dep-dso": [],
+      "dep-dso": [
+        "vendor/lib64/libsdsprpc.so",
+        "vendor/lib/libsdsprpc.so"
+      ],
       "BoardConfigVendor": [],
       "device-vendor": [
         "PRODUCT_PROPERTY_OVERRIDES += \\",
         "    ro.hardware.fingerprint=fpc \\",
+        "    ro.hardware.vulkan=adreno \\",
+        "    ro.hardware.egl=adreno \\",
         "    ro.gfx.driver.0=com.google.pixel.wahoo.gfxdrv \\",
         "    ro.oem_unlock.pst=/dev/block/platform/soc/1da4000.ufshc/by-name/misc \\",
-        "    ro.oem_unlock.pst_offset=6144"
+        "    ro.oem_unlock.pst_offset=6144\\",
```

Comment:
missing space, should be:
`"    ro.oem_unlock.pst_offset=6144 \\",`
for consistency

Reply:
Fixed